src/main/java/uk/gov/hmcts/darts/common/repository/ObjectAdminActionRepository.java

```diff
@@ -34,6 +34,8 @@ Optional<ObjectAdminActionEntity> findByTranscriptionDocument_IdAndObjectHiddenR
     @Query("""
         SELECT o FROM ObjectAdminActionEntity o
                 WHERE o.markedForManualDelDateTime < :deletionThreshold
+                AND ((o.media is not null and o.media.isDeleted = false)
```

Comment:
There are 2 other documents case document and annotation document. Should these also have the same conditions or are they not deleted?